src/pages/pt-BR/guides/aliases.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Atalhos
+description: Uma introdução aos atalhos com Astro.
+---
+
+Um **alias** é uma maneira de criar atalhos para os seus imports.
```

Comment:
I think we can translate this `imports` since we are not exactly talking about the keyword

```suggestion
Um **alias** é uma maneira de criar atalhos para as suas importações.
```